rfq/oracle.go

```diff
@@ -209,6 +225,21 @@ func insecureServerDialOpts() ([]grpc.DialOption, error) {
 		insecure.NewCredentials(),
 	))
 
+	// Configure client-side keepalive parameters. These settings ensure
+	// the client actively probes the connection health and prevents idle
+	// connections from being silently closed by intermediaries or the
+	// server.
+	opts = append(opts, grpc.WithKeepaliveParams(keepalive.ClientParameters{
```

Comment:
nit: Do we want this to be a pure copy-paste of the opts on lines 200-214?  Can't we reuse?